From 42bc99704fe85741f8643f1eb0eaaa6ff0e8d56b Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 25 Oct 2021 17:49:41 +0200
Subject: [PATCH 451/478] arm64: dts: rk3399-pinephone-pro: Correct the battery
 specification
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

XLS with ZCV measurements from the manufacturer say that at 25°C,
ZCV is ~200 mOhm. At 50°C it's at ~100 mOhm. Set some value in between.

Lower the charging current to something less aggressive.

Fix the current sensing resitor value (it's 10 mOhm in the schematic).

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index e6aac03e1ec9..ffdaaa6c7a11 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -504,16 +504,16 @@ battery {
 				4110 4161 4217 4308>;
 			design_capacity = <2916>;
 			design_qmax = <2708>;
-			bat_res = <65>;
+			bat_res = <150>;
 			max_input_current = <3000>;
-			max_chrg_current = <3000>;
+			max_chrg_current = <2000>;
 			max_chrg_voltage = <4350>;
 			sleep_enter_current = <300>;
 			sleep_exit_current = <300>;
 			power_off_thresd = <3400>;
 			zero_algorithm_vol = <3950>;
 			fb_temperature = <105>;
-			sample_res = <20>;
+			sample_res = <10>;
 			max_soc_offset = <60>;
 			energy_mode = <0>;
 			monitor_sec = <5>;
-- 
2.35.3

